app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,43 @@
+class GroupsController < ApplicationController
+  load_and_authorize_resource
+
+  before_action do
+    @employees = User.accessible_by(current_ability)
+  end
+
+  def new
+  end
+
+  def create
+    @group.company = @current_company
```

Comment:
we only need company_id during save and write a method in model to save object (model responsibility)